src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -34,7 +34,77 @@
 final class PhpUnitDedicateAssertFixer extends AbstractPhpUnitFixer implements ConfigurableFixerInterface
 {
     /**
-     * @var array<string, array{positive: string, negative: false|string, argument_count?: int, swap_arguments?: true}|true>
+     * @var array{
```

Comment:
I would like to have this change reverted.
it's giving extra knowledge of "method X will / will not have those keys", while in fact we do not rely on it anywhere - we rely on contract that is "any method, if configured into array, will have those key obligatory and those keys optional" - for whole algo it doesn't matter which method will have which configuration.

then, you also changed configuration from `swap_arguments: true` to `swap_arguments: bool`, which is extending the type (claiming it can take `false` as value, which is not supported). same for changes for `negative`